homeassistant/components/sensor/mercedesme.py

```diff
@@ -0,0 +1,98 @@
+"""
+Support for Mercedes cars with Mercedes ME.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/mercedesme/
+"""
+import logging
+import datetime
+
+from homeassistant.helpers.entity import Entity
+
+DEPENDENCIES = ['mercedesme']
+
+DATA_MME = 'mercedesme'
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Setup the sensor platform."""
+    if discovery_info is None:
+        return
+
+    controller = hass.data[DATA_MME]['controller']
+
+    if not controller.cars:
+        return
+
+    for car in controller.cars:
+        add_devices([Sensor('fuelLevelPercent', car, controller, "%")], True)
+        add_devices([Sensor('fuelRangeKm', car, controller, "Km")], True)
+        add_devices([Sensor('serviceIntervalDays',
+                            car, controller, "days")], True)
+        add_devices([Sensor('odometerKm', car, controller, "Km")], True)
+        add_devices([Sensor('latestTrip', car, controller, None)], True)
+
+
+class Sensor(Entity):
+    """Representation of a Sensor."""
+
+    def __init__(self, SensorName, Car, Controller, Unit):

```

Comment:
Same here. Use snake case and prefix with only one underscore for private attributes.